pkg/pillar/nireconciler/linux_config.go

```diff
@@ -1107,20 +1114,25 @@ func (r *LinuxNIReconciler) getIntendedDnsmasqCfg(niID uuid.UUID) (items []dg.It
 		// Not running DHCP and DNS servers inside EVE for Switch network instances.
 		return
 	}
-	_, bridgeIP, _, _, err := r.getBridgeAddrs(niID)
+	_, bridgeIPs, _, _, err := r.getBridgeAddrs(niID)
 	if err != nil {
 		r.log.Errorf("%s: getIntendedDnsmasqCfg: getBridgeAddrs(%s) failed: %v",
 			LogAndErrPrefix, niID, err)
 		return nil
 	}
+	var bridgeIP net.IP
+	if len(bridgeIPs) != 0 {
+		// For local NI there will be exactly one bridge IP.
```

Comment:
Where do we ensure that there will be only one? Should we check and log or return error if len > 1?